openreview/tools.py

```diff
@@ -216,7 +217,16 @@ def capitalize_title(title):
         authors = 'Anonymous'
     else:
         first_author_last_name = note.content['authors'][0].split(' ')[-1].lower()
-        authors = ' and '.join(note.content['authors'])
+        if names_reversed:
```

Comment:
Pam, could you validate this output in one of our unit test?

For example, after we create a paper here:

https://github.com/iesl/openreview-py/blob/master/tests/test_single_blind_conference.py#L175